dataprofiler/data_readers/csv_data.py

```diff
@@ -220,7 +225,7 @@ def _guess_header_row(data_as_str, suggested_delimiter=None,
                 len_increase = True
                 
             # Returns the last row that could reasonably be the header
-            if (rstr > str_thresh and none <= none_thresh and diff <= diff_thresh):
+            if (rstr > str_thresh and none < none_thresh and diff < diff_thresh):
```

Comment:
Set to non-inclusive, just matches what I'd expect as a "threshold". For instance if there were 4 columns, we'd expect >2 columns to have header if we are to say a header exists at all.

Reply:
i think keeping <= makes more sense, but we can always change the thresholds

Reply:
I guess I view thresholds as max limits, i.e. you cannot go beyond that. Setting to be inclusive doesn't seem appropriate as it's "on the threshold". If we want to change it, that's probably fine, but then we should change the input constraints. 

Personally, I think this is opinion more than fact, I just wanted to hold it consistent across the board (non-inclusive).